docs-ref-mapping/reference-unified.yml

```diff
@@ -917,6 +922,7 @@
       uid: azure.nodejs.sdk.landingpage.services.metricsadvisor
       href: ~/docs-ref-services/{moniker}/ai-metrics-advisor-readme.md
       children:
+      - azure-ai-metrics-advisor
```

Comment:
I don't think having this present will negatively affect you, but I'm going to let the current publishing build run before merging to be certain.

Reply:
My concern is that these are extraneous, as we already pull this with the entry for `@azure/metrics-advisor`